shift-react-app/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@binaris/shift-react-app",
+  "version": "0.0.1",
+  "keywords": [
+    "shiftjs",
+    "react"
+  ],
+  "description": "Shift React App",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/binaris/shiftjs.git",
+    "directory": "shift-react-app"
+  },
+  "bugs": {
+    "url": "https://github.com/binaris/shiftjs/issues"
+  },
+  "bin": {
+    "shift-react-app": "./index.js"
```

Comment:
Why do we need `index.js`? We could write the entire thing in typescript

Reply:
You're right..
In concord I made a `bin` directory and created a JS file called `concord` that has the shebang.
Your version is good as well.